release-tools/SIDECAR_RELEASE_PROCESS.md

```diff
@@ -100,7 +100,7 @@ naming convention `<hostpath-deployment-version>-on-<kubernetes-version>`.
 1. If release was a new major/minor version, create a new `release-<minor>`
    branch at that commit.
 1. Check [image build status](https://k8s-testgrid.appspot.com/sig-storage-image-build).
-1. Promote images from k8s-staging-sig-storage to k8s.gcr.io/sig-storage. From
+1. Promote images from k8s-staging-sig-storage to registry.k8s.io/sig-storage. From
```

Comment:
Remove this change. "release-tools" is read-only in this repo, such a change needs to go into csi-release-tools.

Reply:
Removed.